slides/src/content/display.md

```diff
@@ -0,0 +1,148 @@
+<!-- .slide: data-background="../content/images/title-slide.jpg" -->
+
+## Building Applications with Angular
+
+# Displaying Data
+
+---
+
+## Make the Data
+
+- Change the title
+- Add a list of strings
+
+_src/app/app.component.ts_
+```ts
+@Component({
+  …as before…
+}
+export class AppComponent {
+  title = 'To Do';
+  thingsToDo = [
+    'Learn JavaScript',
+    'Learn Angular',
+    'Learn Redux'
+  ];
+}
+```
+
+---
+
+## Display the Data
+
+- Use `*ngFor` to repeat some HTML multiple times
+- The loop variable `item` is created locally
+
+_src/app/app.component.html_
+```html
+<h1>{{title}}</h1>
+<p *ngFor="let item of thingsToDo">{{item}}</p>
+```
+
+![ngFor Output](content/images/screenshot-ngfor.png)
+
+---
+
+## Make Items Findable
+
+- Giving each item a unique ID allows bookmarking and testing
+
+_src/app/app.component.html_
+```html
+<h1>{{title}}</h1>
+<p *ngFor="let item of thingsToDo; let i = index" id="{{i}}">({{i}}) {{item}}</p>
```

Comment:
This code produces the "(1) Learn JavaScript" output but is not reflected in the repo code. These should be synchronized and either all show indices or none. ? Maybe? Unless the training narrative somehow addresses this...